etc/emqx.conf

```diff
@@ -459,6 +459,12 @@ acl_cache_ttl = 1m
 ## MQTT Protocol
 ##--------------------------------------------------------------------
 
+## Response Topic Prefix
+##
+## Value: String
+## Default: 1MB
+mqtt.response_topic_prefix = ResponseTopic
```

Comment:
Since response information is non-standard.
perhaps it makes more sense to have a emqx unique prefix?
so developers for different clients may have a choice to tell from the prefix which server they are working with.
e.g. `emqxrspv1`